desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -111,7 +112,7 @@ MainTab::MainTab(QWidget *parent) : QTabWidget(parent),
 	// This needs to be the same order as enum dive_comp_type in dive.h!
 	QStringList types = QStringList();
 	for (int i = 0; i < NUM_DIVEMODE; i++)
-		types.append(QString(tr(divemode_text_ui[i])));
+		types.append(QString(gettextFromC::instance()->tr(divemode_text_ui[i])));
```

Comment:
Here too: No `QString` needed.

Reply:
is QString is really needed, that is something that calls for a comment in the code, otherwise the next maintainer will remove it.

btw. why is needed ?